torch/quantization/_equalize.py

```diff
@@ -0,0 +1,130 @@
+import torch
+import copy
+
+def max_over_ndim(input, axis_list, keepdim=False):
+    ''' Applies 'torch.max' over the given axises
+    '''
+    axis_list.sort(reverse=True)
+    reduced_input = input
+    for axis in axis_list:
+        reduced_input, _ = reduced_input.max(axis, keepdim)
+    return reduced_input
+
+def min_over_ndim(input, axis_list, keepdim=False):
+    ''' Applies 'torch.min' over the given axises
+    '''
+    axis_list.sort(reverse=True)
+    reduced_input = input
+    for axis in axis_list:
+        reduced_input, _ = reduced_input.min(axis, keepdim)
+    return reduced_input
+
+def channel_range(input, axis=0):
+    ''' finds the range of weights associated with a specific channel
+    '''
+    size_of_tensor_dim = len(input.size())
+    mins = input
+    maxs = input
+
+    axis_list = list(range(size_of_tensor_dim))
+    axis_list.remove(axis)
+
+    mins = min_over_ndim(mins, axis_list)
+    maxs = max_over_ndim(maxs, axis_list)
+
+    assert(mins.size()[0] == input.size()[axis])
```

Comment:
Can you, please, add a comment here telling the user why the error happened? Also, note that the parenthesis in assertions are dangerous :) Sauce: https://stackoverflow.com/a/3112178

```python
    assert mins.size()[0] == input.size()[axis], "Some useful error message here..."
```